src/io/der/octet_aligned.rs

```diff
@@ -2,6 +2,126 @@ use crate::io::buf::OctetBuffer;
 use crate::io::der::Error;
 use crate::io::der::{DistinguishedRead, DistinguishedWrite};
 
-impl DistinguishedRead for OctetBuffer {}
+#[derive(Debug)]
+pub enum Class {
```

Comment:
Just saw there is an implementation for TAG, Type and Size in `asn1rs-model/src/model/mod.rs`.
I think it's better that I use them, then you provide them in `C` for each parser function, so I can validate them

Reply:
My bad. You just implemented it in [cfb2101](https://github.com/kellerkindt/asn1rs/commit/cfb21015ccb255e7032b2dc1eca5f845e88ffe1b).
The code is a little fragmented for now, I think I need to wait for you to merge all of them, then we continue on top of that

Reply:
I just rebased the `experimental-set-setof` branch to include the changes of cfb2101.
I`ll probably merge it into master tomorrow (~20-ish hours from now).